web-components/src/components/floating-modal/scss/floating.scss

```diff
@@ -26,18 +33,94 @@
     justify-content: space-between;
     max-height: $floating-header-max-height;
     padding: $floating-header-padding;
+    width: 100%;
+
+    &-text {
+      flex: 2 1 auto;
+      font-family: $body-font-family;
+      font-size: $font-size-large;
+      line-height: $font-size-large;
+      text-align: left;
+    }
+  }
+
+  .md-floating__body {
+    height: calc(100% - 3rem);
+    overflow: auto;
+    padding: $floating-body-padding;
+  }
+
+  
+}
+
+.md-floating-min {
+  background-color: var(--floating-modal-main-bg-color, $lm-floating-modal-main-bg-color-alternate-light);
+  border: $floating-main-border;
+  border-radius: 8px;
+  box-shadow: var(--floating-modal-main-box-shadow, $lm-floating-modal-main-box-shadow-light);
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+  max-width: 100%;
+  position: absolute;
+  touch-action: none;
+  user-select: none;
+  z-index: 1040;
+
+  &.hide {
```

Comment:
there are duplicate styles? is there a way to reuse?

Reply:
Done, remvoed all duplicated styles